internal/capability/report.go

```diff
@@ -0,0 +1,77 @@
+package capability
+
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+func (ca CapAudit) Report(opts ...ReportOption) error {
+
+	for _, opt := range opts {
+
+		err := opt.report(ca)
+		if err != nil {
+			logger.Debugw("Unable to generate report for %T", opt, "Error: %s", err)
```

Comment:
Could you change this to Debugf instead of Debugw. Debugw will not accept this format.

Reply:
done!